res/drawable/overflow.xml

```diff
@@ -0,0 +1,10 @@
+<!-- drawable/dots_vertical.xml -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
```

Comment:
I played with this and I think it would look better when using 18dp size here and using android:scaleType="center" in the ImageView displaying it. With the current size IMHO it looks a bit large.

Thoughts?